src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -631,6 +632,13 @@
     AttributeModifierBuilder getAttributeModifierBuilder();
 
     /**
+     * Gets a new {@link AttributeCalculator}.
+     * 
+     * @return A new {@link AttributeCalculator}
+     */
+    AttributeCalculator getAttributeCalculator();
```

Comment:
Does this really returns a new instance?
I guess it should return the registered/default `AttributeCalculator`.

Because the current interface looks really stateless.

Or is there a reason you always need a new instance.
In which cases? One per entity? One per calculation?